ds701_book/25-NN-III-CNNs.qmd

```diff
@@ -372,124 +377,121 @@ Problems with fully-connected networks
 * Should be stable under transformations
     * Don’t want to re-learn appearance at different parts of image
 
+## Classification Invariant to Shift
+
 <!-- Image Credit "https://udlbook.github.io/udlbook/"-->
 
-#### Classification Invariant to Shift
+![](figs/NN-figs/L25-shift-img-class.png){width="75%" fig-align="center"}
```

Comment:
Can you add a sentence to explain what this figure is intended to convey?

Reply:
done in next commit.